app/ui/src/main/res/values-br/strings.xml

```diff
@@ -102,8 +102,8 @@ Danevellit beugoù, kenlabourit war keweriusterioù nevez ha savit goulennoù wa
   <string name="forward_action">Treuzkas</string>
   <string name="forward_as_attachment_action">Treuzkas evel stagadenn</string>
   <string name="move_action">Dilec’hiañ</string>
-  <string name="single_message_options_action">Kas...</string>
-  <string name="refile_action">Kemman ar rummad...</string>
+  <string name="single_message_options_action">Kas…</string>
```

Comment:
We manage translations on Transifex: https://www.transifex.com/k-9/k9mail/
Changes to the files in the repository will be overwritten when translations are pulled from transifex. Please revert changes to all `strings.xml` versions except the one in `values`.

But I'd appreciate these fixes be made on Transifex instead.